src/scripts/getGif.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosResponse } from "axios";
+
+import api from "./api";
+
+/**
+ * This is the get gif script, this script will get a list of GIFs
+ * from the GIPHY API via the CHill&chat API.
+ *
+ * @param {string} search The search query.
+ * @param {boolean} useTrending Wether use trending GIFs or not.
+ * @returns {any} Returns a list of either trending or searched GIFs.
+ * @see https://developers.giphy.com/docs/api/schema#gif-object for schema def.
```

Comment:
Replace ```def``` with ```definition``` for more of a formal text

Reply:
Fixed in [2a8ff0e](https://github.com/Chillandchat/mobile/pull/84/commits/2a8ff0e8fcefe6453df22d479f477aeaf8544517)